### DISCUSSION TITLE
Pregabalin + Cannabis

### DISCUSSION THREAD

**Discussion by Jedrek873326** [TripSit/drugs#165] [Drug Interactions]
I've found one study that suggest no risks when mixing pregabalin with cannabis. the study was done to see the effectiveness of using pregabalin with quitting thc
"No severe side effects were noted in either experiment."
Study:
https://pmc.ncbi.nlm.nih.gov/articles/PMC8969895/
Interaction checker on drugs:com
https://www.drugs.com/interactions-check.php?drug_list=2758-0,1937-0
Pregabalin has also been suggested as a treatment of CUD
https://psycnet.apa.org/doiLanding?doi=10.1037%2Fpha0000464

**Comment by zelixir25** [comment]
citation for the study:

Lile, J. A., Alcorn, J. L., Hays, L. R., Kelly, T. H., Stoops, W. W., Wesley, M. J., & Westgate, P. M. (2022). Influence of pregabalin maintenance on cannabis effects and related behaviors in daily cannabis users. Experimental and clinical psychopharmacology, 30(5), 560. https://dx.doi.org/10.1037/pha0000464

sorry updated to have the doi as well

**Comment by Jedrek873326** [comment]
Might be a tiny bit relevant only, but individual subjective effects could help later on 
Source: https://hyperreal.info/talk/pregabalina-miksy-t59471.html
Opinions about this mix from users:
- "increased sedation"
- "drowsiness"
- "no revelations, you're just walking wobbly"
- "(pregab)synergizes with mj very well"
- "btw, pregab synergizes really nicely"
- "Today, after mixing pregabalin with MJ, the letters I was reading started jumping around—a curious effect, I recommend it. And the whole time I feel like I’m literally in a circus; the music, vision, and imbalance are causing this effect."

**Comment by utaninja** [comment]
Apologies for the delay, this totally got buried 😅 .  I actually worked on this interaction way later and came to the same conclusion (https://github.com/TripSit/drugs/issues/218). I've made a new issue to add the hyperreal source as I already had the others. I'm also considering this as a review given we came to the same conclusions (:

You can see the new updated issue here - https://github.com/TripSit/drugs/issues/376

Thanks again and sorry about the delay!
